frontend/packages/dev-console/integration-tests/features/addFlow/create-from-git.feature

```diff
@@ -203,3 +203,13 @@ Feature: Create Application from git form
                   | git_url                                |
                   | https://github.com/sclorg/httpd-ex.git |
                   | https://github.com/sclorg/nginx-ex.git |
+
+        @regression @manual
+        Scenario: Provide custom build environments for nodejs git import
+            Given user is at Import from Git form
+            When user enters Git Repo URL as "https://github.com/sclorg/nodejs-ex"
+            And user enters "NPM_RUN" build environment field as "build2"
```

Comment:
That is not what the user does. The user enters a "Run command", not a build environment variable.

Reply:
Updated.